### ISSUE TITLE
`qisrc init` fails when `manifest_url` is a relative path

### ISSUE THREAD

**Issue by Grimy** [aldebaran/qibuild#27]
I have a `manifest` git repository in the current directory containing a `manifest.xml` file. `qisrc init manifest` fails with the following output:

```
Updating manifest ...
*  (master) 
[ERROR]:  Exception Update failed
git fetch origin failed
fatal: 'manifest' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

This is quite strange, considering that `manifest` is, indeed, a git repository, and that `cd manifest && git fetch origin` works alright.

As a workaround, specifying the full path to the local repository (`qisrc init $PWD/manifest`) works fine.

**Comment by Grimy** [comment]
As another workaround, you can also specify a relative path by prefixing it with `../../../`; in my example, `qisrc init ../../../manifest` works fine (but is ugly).

**Comment by dmerejkowsky** [comment]
Fixed in 3.10: 23bd4142058453c124a51e33b4a2aae016357d04
